draft-ietf-webtrans-http3.md

```diff
@@ -276,27 +276,15 @@ SHALL last until the end of the stream.
 ~~~~~~~~~~
 {: #fig-bidi-client title="WEBTRANSPORT_STREAM frame format"}
 
-## Server-Initiated Bidirectional Streams
+HTTP/3 does not by itself define any semantics for server-initiated
+bidirectional streams.  If WebTransport setting is negotiated by both
+endpoints, the syntax of the server-initiated bidirectional streams SHALL be
+the same as the syntax of client-initated bidirectional streams, that is, a
+sequence of HTTP/3 frames.  The only allowed frame is WEBTRANSPORT_STREAM;
+subsequent documents may add more frames, or define server-initiated semantics
+for existing frames.
 
-WebTransport servers can initiate bidirectional streams by opening a
-bidirectional stream within the HTTP/3 connection.  Note that since HTTP/3 does
-not define any semantics for server-initiated bidirectional streams, this
-document is a normative reference for the semantics of such streams for all
-HTTP/3 connections in which the SETTINGS_ENABLE_WEBTRANSPORT option is negotiated.
-The format of those streams SHALL be the session ID, encoded as a
-variable-length integer, followed by the user-specified stream data
-({{fig-bidi-server}}).
-
-~~~~~~~~~~ drawing
-  0                   1                   2                   3
-  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
-  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-  |                        Session ID (i)                       ...
-  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-  |                         Stream Body                         ...
-  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-~~~~~~~~~~
-{: #fig-bidi-server title="Server-initiated bidirectional stream format"}
+TODO: move the paragraph above into a separate draft.
```

Comment:
I think maybe all you need to say is 

```
The only allowed HTTP/3 frame on a WebTransport bidirectional stream is WEBTRANSPORT_STREAM.
```

I don't see a lot of value here saying that some other document might specify something else.

Reply:
Made text more vague; let's discuss this further on #23 and at the upcoming interim.